sirepo/pkcli/job_agent.py

```diff
@@ -751,6 +751,19 @@ def _processor():
 if [[ ! $LD_LIBRARY_PATH =~ /usr/lib64/mpich/lib ]]; then
```

Comment:
This probably isn't needed any more?

Reply:
It isn't https://github.com/radiasoft/sirepo/pull/4357/commits/87ef691b2e109cfb81600600cc11a39568aab666